docs/data/material/components/buttons/DisableElevation.js

```diff
@@ -1,10 +1,14 @@
 import * as React from 'react';
+import Stack from '@mui/material/Stack';
```

Comment:
We can't do this yet, but this could make a lot of sense for v6 #40594.

```suggestion
import Stack from '@mui/system/Stack';
```